packages/block-editor/src/components/block-content-overlay/index.js

```diff
@@ -88,7 +103,9 @@ export default function BlockContentOverlay( {
 			onMouseEnter={ () => setIsHovered( true ) }
 			onMouseLeave={ () => setIsHovered( false ) }
 			onMouseUp={
-				isOverlayActive ? () => setIsOverlayActive( false ) : undefined
+				isOverlayActive && canEdit
```

Comment:
Is `canEdit` needed here? The `useEffect` above seems to set `isOverlayActive` back to `true` even if it becomes `false` here. Or is this check meant to prevent this tiny blip of `isOverLayActive` being set to false and then right away to true?

Reply:
> Or is this check meant to prevent this tiny blip of isOverLayActive being set to false and then right away to true

Correct. It's not always noticeable, but there's also no need to attach an event callback when editing is disabled.